docs/pages/guides/customizing-webpack.mdx

```diff
@@ -75,7 +75,7 @@ To get everything working properly, you'll want to install and include a global
 ### Adding ResizeObserver
 
 - Install the polyfill:
-  <Terminal cmd={['$ npm install resize-observer-polyfill']} />
+  <Terminal cmd={['$ yarn add resize-observer-polyfill']} />
```

Comment:
```suggestion
  <Terminal cmd={['$ npx expo install resize-observer-polyfill']} />
```

Let's recommend `expo install` instead. It will use npm or Yarn based on the user and project settings.